src/report_tests.py

```diff
@@ -65,6 +67,23 @@ def append(self, status: str, test: str, comment: str = '') -> None:
     if summary.text == '':
         comment = f'No tests ran on {machine_name()}.'
     else:
-        comment = summary.text
+        summary_rows = list(filter(None, summary.text.split('\n')))
+        logfiles = []
+        col = []
+        for item in summary_rows:
+            logfiles.append(item[item.find('<a'):item.rfind('a>') + 2])
+            col.append(item.split()[0])
+
+        comment = f'###  {machine_name()}\n'
```

Comment:
`###` is markdown notation for a header of level 3.
Maybe extend `Markdown` with general purpose functions `header_1` to `header_6(text: str) -> str:`